File: python-example/pages/customer_list_page.py
from selenium.webdriver.support.wait import WebDriverWait


class CustomerListPage:

    def __init__(self, driver):
        self.driver = driver
        self.wait = WebDriverWait(driver, 10)

    def open(self):
        self.driver.get("http://localhost/litecart/admin/?app=customers&doc=customers")
        return self

    @property
    def customer_rows(self):
        return self.driver.find_elements_by_css_selector("table.dataTable tr.row")

    def get_customer_ids(self):
        return set([e.text for e in self.customer_rows])

File: csharp-example/csharp-example/pages/CustomerListPage.cs
using System.Linq;
using System.Collections.Generic;
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;

namespace csharp_example
{
    internal class CustomerListPage : Page
    {
        public CustomerListPage(IWebDriver driver) : base(driver)
        {
            PageFactory.InitElements(driver, this);
        }

        internal CustomerListPage Open()
        {
            driver.Url = "http://localhost/litecart/admin/?app=customers&doc=customers";
            return this;
        }

        [FindsBy(How = How.CssSelector, Using = "table.dataTable tr.row")]
        IList<IWebElement> customerRows;

        internal ISet<string> GetCustomerIds()
        {
            return new HashSet<string>(
                customerRows.Select(e => e.FindElements(By.TagName("td"))[2].Text).ToList());
        }
    }
}